src/libs/EmojiUtils.ts

```diff
@@ -414,23 +393,25 @@ function suggestEmojis(text, lang, limit = CONST.AUTO_COMPLETE_SUGGESTER.MAX_AMO
         return [];
     }
 
-    const matching = [];
+    const matching: Emoji[] = [];
     const nodes = trie.getAllMatchingWords(emojiData[0].toLowerCase().slice(1), limit);
-    for (let j = 0; j < nodes.length; j++) {
-        if (nodes[j].metaData.code && !_.find(matching, (obj) => obj.name === nodes[j].name)) {
+    for (const node of nodes) {
+        if (node.metaData.code && !matching.find((obj) => obj.name === node.name)) {
             if (matching.length === limit) {
                 return matching;
             }
-            matching.push({code: nodes[j].metaData.code, name: nodes[j].name, types: nodes[j].metaData.types});
+            matching.push({code: node.metaData?.code ?? '', name: node.name, types: node.metaData.types});
```

Comment:
Already have this check - `node.metaData.code && ` in if condition
```suggestion
            matching.push({code: node.metaData.code, name: node.name, types: node.metaData.types});
```